package.json

```diff
@@ -71,10 +71,9 @@
     "clean-webpack-plugin": "3.0.0",
     "dotenv": "8.2.0",
     "dotenv-webpack": "3.0.0",
-    "html-webpack-new-relic-plugin": "2.0.0",
     "html-webpack-plugin": "4.5.0",
     "mini-css-extract-plugin": "0.11.2",
-    "webpack": "4.44.2",
+    "webpack": "^5.0.0",
```

Comment:
🎉  Lol, this probably should have been bumped when we upgraded to the version of `@edx/frontend-build` that introduced Webpack 5 in the first place. Thanks for addressing!